src/WhatToCook.Application/Domain/PlanOfmeals.cs

```diff
@@ -2,9 +2,19 @@
 
 public class PlanOfMeals
 {
+    public string Name { get; set; }
     public int Id { get; private set; }
     public DateTime FromDate { get; set; }
     public DateTime ToDate { get; set; }
-    public Recipe Recipe { get; set; }
-    public User User { get; set; }
+    public User User { get; set; } = new User() { Email = "mail123@gmail.com" };
+    public IEnumerable<Recipe> Recipes { get; set; }
+
+    public PlanOfMeals(string name, DateTime fromDate, DateTime toDate, IEnumerable<Recipe> recipes)
+    {
+        Name = name;
+        FromDate = fromDate;
```

Comment:
Validation can be added, similar to the one we have on the frontend

Reply:
added validation and tests to check if it works, will be in the next commit